.github/ISSUE_TEMPLATE/c-tom-md.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Cútom.md
+about: spring-boot-web
+
+---
+
+GitHub.com uses its own version of the Markdown syntax that provides an additional set of useful features, many of which make it easier to work with content on GitHub.com.
+
+Note that some features of GitHub Flavored Markdown are only available in the descriptions and comments of Issues and Pull Requests. These include @mentions as well as references to SHA-1 hashes, Issues, and Pull Requests. Task Lists are also available in Gist comments and in Gist Markdown files.
+
+Syntax highlighting
+
+Here’s an example of how you can use syntax highlighting with GitHub Flavored Markdown:
+
+```javascript function fancyAlert(arg) { if(arg) { $.facebox({div:'#foo'}) } } ``` 
```

Comment:
On Jan 16, 2019 7:05 PM, "prologic-dev" <notifications@github.com> wrote: command type: npm start below is the output 0 info it worked if it ends with ok 1 verbose cli [ '/home/prologic/.nvm/versions/node/v8.0.0/bin/node', 1 verbose cli '/home/prologic/.nvm/versions/node/v8.0.0/bin/npm', 1 verbose cli 'run', 1 verbose cli 'ionic:serve' ] 2 info using npm@5.0.0 3 info using node@v8.0.0 4 verbose run-script [ 'preionic:serve', 'ionic:serve', 'postionic:serve' ] 5 info lifecycle copay@5.4.2preionic:serve: copay@5.4.2 6 silly lifecycle copay@5.4.2preionic:serve: no script for preionic:serve, continuing 7 info lifecycle copay@5.4.2ionic:serve: copay@5.4.2 8 verbose lifecycle copay@5.4.2ionic:serve: unsafe-perm in lifecycle true 9 verbose lifecycle copay@5.4.2ionic:serve: PATH: /home/prologic/.nvm/versions/node/v8.0.0/lib/node_modules/npm/bin/node-gyp-bin:/var/www/html/work/wallet/copay/node_modules/.bin:/home/prologic/.nvm/versions/node/v8.0.0/lib/node_modules/npm/bin/node-gyp-bin:/var/www/html/work/wallet/copay/node_modules/.bin:/home/prologic/.nvm/versions/node/v8.0.0/bin:/home/prologic/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin 10 verbose lifecycle copay@5.4.2ionic:serve: CWD: /var/www/html/work/wallet/copay 11 silly lifecycle copay@5.4.2ionic:serve: Args: [ '-c', 'ionic-app-scripts serve --nolivereload' ] 12 info lifecycle copay@5.4.2ionic:serve: Failed to exec ionic:serve script 13 verbose stack Error: copay@5.4.2 ionic:serve: ionic-app-scripts serve --nolivereload 13 verbose stack spawn ENOENT 13 verbose stack at ChildProcess. (/home/prologic/.nvm/versions/node/v8.0.0/lib/node_modules/npm/lib/utils/spawn.js:33:16) 13 verbose stack at emitTwo (events.js:125:13) 13 verbose stack at ChildProcess.emit (events.js:213:7) 13 verbose stack at maybeClose (internal/child_process.js:887:16) 13 verbose stack at Process.ChildProcess._handle.onexit (internal/child_process.js:208:5) 14 verbose pkgid copay@5.4.2 15 verbose cwd /var/www/html/work/wallet/copay 16 verbose Linux 4.15.0-20-generic 17 verbose argv "/home/prologic/.nvm/versions/node/v8.0.0/bin/node" "/home/prologic/.nvm/versions/node/v8.0.0/bin/npm" "run" "ionic:serve" 18 verbose node v8.0.0 19 verbose npm v5.0.0 20 error file sh 21 error code ELIFECYCLE 22 error errno ENOENT 23 error syscall spawn 24 error copay@5.4.2 ionic:serve: ionic-app-scripts serve --nolivereload 24 error spawn ENOENT 25 error Failed at the copay@5.4.2 ionic:serve script. 25 error This is probably not a problem with npm. There is likely additional logging output above. 26 verbose exit [ 1, true ] Failed to exec ionic:serve script 13 verbose stack Error: copay@5.4.2 ionic:serve: ionic-app-scripts serve --nolivereload — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub <#9472>, or mute the thread <https://github.com/notifications/unsubscribe-auth/Araitz3NBUOoQ2Oh6ELAWYndoVSPRCRPks5vDweHgaJpZM4aC0v6> .